css/global.scss

```diff
@@ -0,0 +1,65 @@
+$primaryColor: #4AB777;
```

Comment:
You should split global styles and variables so you don't need to `@import "global";` in other files.